documentation/latest/operations/deployment_options.md

```diff
@@ -79,3 +79,81 @@ no need to have human interaction to start running workloads on any device.
 - **Default**: :9443
 - **Values**: __port range__
 - **Description**: WebhookPort is the port that the webhook server serves at.
+
+# Edge API Deployment options
+
+Edge API has different deployment modes, where some parameters can be turned
+on/off based on a configMap value. To edit one of these parameters, the
+following configmap should be edited.
+
+```shell
+kubectl edit cm -n flotta flotta-edge-api-config
+```
+
+After any change in that file, flotta-operator and API need to be rebooted.
```

Comment:
Why a reboot is needed for flotta-operator?

Reply:
:facepalm: copy & paste. On it.

Reply:
Fixed.